components/form/Probe.vue

```diff
@@ -302,12 +302,13 @@ export default {
 </template>
 
 <style lang="scss" scoped>
+.title {
+  margin-bottom: 10px;
+}
 
-  .title {
-    margin-bottom: 10px;
+::v-deep .labeled-select {
+  &:not(taggable) {
+    height: auto !important;
```

Comment:
Can we solve this issue without relying on `!important`? Using `!important` is generally considered bad practice and I'd like to see if we can apply the style via specificity first.

Reply:
I agree, not a fan of using `!important` if it isn't necessary. Are you opposed to me changing the line that is [found here](https://github.com/rancher/dashboard/blob/aeceab87431991087ddad6a706248eda76dc39d7/assets/styles/global/_select.scss#L16)? That seems to be the root cause that's affecting LabeledSelect everywhere.

Reply:
As we finally changed all the inputs for the ticket, you can @jordojordo simply revert the change for the [faulty commit](https://github.com/rancher/dashboard/pull/5071/commits/b26a88ea4093bf4df59028e257576bccc5ed06de) (ops :grimacing: ), as it is not required anymore.

I also agree with @rak-phillip, we definitely need to refactor all the styling done for all the types of select we have, since there's multiple cases, also we reference other elements as [in this case](https://github.com/rancher/dashboard/blob/master/assets/styles/global/_select.scss) for the `.col`.